jest.js

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line import/no-commonjs
+module.exports = {
+  env: {
+    jest: true,
+  },
+  extends: ['plugin:jest/recommended', './base.js'],
```

Comment:
Safest way is to try it actually, what I do is publish a beta version with my changes to eslint-config-algolia locally (on a @next tag).

Then use it inside a project and ensure the behavior is the right one (prettier is active and well working).

Once I have it, I'll just do a PR. I think I did that to test this actually and that's why it's made like that. But unsure so please doublecheck